core/src/models/Export.ts

```diff
@@ -293,10 +293,8 @@ export class Export extends Model {
   }
 
   @BeforeCreate
-  static generateId(instance) {
-    if (!instance.id) {
-      instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
-    }
+  static generateId(instance: Export) {
+    if (!instance.id) instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
```

Comment:
Why do these keep happening. Some sort of linter?

Reply:
fwiw and i don't feel that strongly about it, but I think the braces prevent mistakes even though one line looks nicer.

Reply:
Nah, it's me manually doing it with a find/replace when I find some duplicated code to clean up.  Prettier seems to be OK with it, and I find them much easier to read.